### PR TITLE
add fps collecting to tests

### PR DESCRIPTION
**Pull request by ivanvikhrev**
Some demos output FPS to console, the change allows to collect this info from logs and store to report during tests.

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/tests/run_tests.py [hunk 1]
```diff
@@ -211,17 +214,23 @@ def option_to_args(key, value):
...
                             start_time = timeit.default_timer()
-                            subprocess.check_output(fixed_args + dev_arg + case_args,
+                            demo_output = subprocess.check_output(fixed_args + dev_arg + case_args,
                                 stderr=subprocess.STDOUT, universal_newlines=True)
                             execution_time = timeit.default_timer() - start_time
+                            match_fps = fps_finder.search(demo_output)
+                            if match_fps is not None:
+                                fps = match_fps.group(0)
+                            else:
+                                fps = 'N/A'
```
Comment (eizamaliev) [inline demos/tests/run_tests.py:224]:
```suggestion
                            fps = match_fps.group(0) if match_fps else 'N/A'
```

**Review by eizamaliev — CHANGES_REQUESTED** [review]

File: demos/tests/args.py [hunk 2]
```diff
@@ -40,6 +40,26 @@ def brats_arg(id):
 def image_retrieval_arg(id):
     return TestDataArg('Image_Retrieval/{}'.format(id))
 
+class NumThreads:
```
Comment (eizamaliev) [inline demos/tests/args.py:43]: What is the purpose of such classes?
Could we specify `single_option_cases('-nthreads', 1, 2, 4, 8),` or not?

**Comment by ivikhrev** [comment]
jenkins ok to test

**Comment by vladimir-dudnik** [comment]
12:50:08  demos/tests/args.py:58: trailing whitespace.

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Pull request closed without merging**